src/Functions/FunctionsHashing.cpp

```diff
@@ -2,6 +2,13 @@
 
 #include <Functions/FunctionFactory.h>
 
+/// FunctionsHashing are separated into files:
```

Comment:
```suggestion
/// FunctionsHashing instantiations are separated into files:
```

Reply:
Actually, this file (FunctionsHashing.cpp) does the same thing as FunctionsHashingInt/Murmur/SSL.cpp (i.e. instantiate `registerFunction<>()`). For symmetry reasons, we should maybe
- rename this file to "FunctionsHashingMisc.cpp" and
- put similar comments as the one on l. 5 into the other instantiating files? (also free to replace l. 6 - l. 9 by a simple `- Functions*.cpp`. Who knows how many more splits we'll need 😄